src/components/PendingAppList.js

```diff
@@ -1,7 +1,6 @@
-import React from "react"
+import React, { useState } from "react"
```

Comment:
Where in this component are you using useState? If it is not being declared in this component it isn't needed.

Reply:
updated to remove for now.  Was planning on using it but ended up not.  May add back in later